tokens/movistar.json

```diff
@@ -1304,9 +1304,9 @@
       "description": "pepper45"
     },
     "tagBackgroundPromo": {
-      "value": "{palette.darkModeGrey7}",
+      "value": "{palette.pepper55}",
```

Comment:
Why `pepper55` and not d`arkModeGrey7` that aligns with the other tags in dark?

Reply:
Because the new Promo style is far away from the rest of the tags